hw/xbox/nv2a/swizzle.c

```diff
@@ -131,9 +144,14 @@ void unswizzle_box(
     for (z = 0; z < depth; z++) {
         for (y = 0; y < height; y++) {
             for (x = 0; x < width; x++) {
-                const uint8_t *src = src_buf
-                    + get_swizzled_offset(x, y, z, mask_x, mask_y, mask_z,
-                                          bytes_per_pixel);
+                const uint8_t *src = src_buf +
+                                     get_swizzled_offset(x,
+                                                         y,
+                                                         z,
+                                                         mask_x,
+                                                         mask_y,
+                                                         mask_z,
+                                                         bytes_per_pixel);
```

Comment:
Harder to read now.

Reply:
Need to double check line limit, if it's < 80 then I'm ok to put them back on one line